### PR TITLE
New addon: echo-effect

### PR DESCRIPTION
**Pull request by apple502j**
See #1811 (and yes, echo effect is not removed from code, yet.)

### CODE REVIEW

File: addons-l10n/en/echo-effect.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "echo-effect/echo": "Echo"
```
Comment (TheColaber) [inline addons-l10n/en/echo-effect.json:2]: Did they remove the msg but not the code?

File: addons/echo-effect/addon.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Echo effect button",
+  "description": "Brings back the Echo button in the sound editor.",
+  "tags": ["editor"],
```
Comment (TheColaber) [inline addons/echo-effect/addon.json:4]: dont forget to add versionAdded and maybe make this featured?

**Review by TheColaber — CHANGES_REQUESTED** [review]

File: addons-l10n/en/echo-effect.json (hunk 1 above)
Comment (apple502j) [reply addons-l10n/en/echo-effect.json:2]: Hmm, seems like "yes".

**Comment by WorldLanguages** [comment]
Anyone know why it was removed?

**Comment by CST1229** [comment]
> Anyone know why it was removed?

IIRC it was removed because there were too many buttons in the sound editor, and it needed a bit of cleanup. The echo effect was the one that was decided to go.

**Comment by mxmou** [comment]
> > Anyone know why it was removed?
> 
> IIRC it was removed because there were too many buttons in the sound editor, and it needed a bit of cleanup. The echo effect was the one that was decided to go.

[This post](https://scratch.mit.edu/discuss/post/5271499) by a Scratch Team member confirms that. Interestingly, the [update announcement](https://scratch.mit.edu/discuss/topic/365737/) doesn't mention echo at all, and [the PR](https://github.com/LLK/scratch-gui/pull/4974) doesn't include the reason either.

**Comment by lisa-wolfgang** [comment]
If the developers really wanted to clean up the sound editor, removing just one button while adding many more wasn't the way to do it.

**Comment by WorldLanguages** [comment]
Ok, good. I wanted to make sure it wasn't removed for some weird reason, like idk, using this effect too many times causing a ear rape sound idk :P

**Pull request merged**